client/blocks/plugins-scheduled-updates/schedule-form-paths.tsx

```diff
@@ -5,10 +5,13 @@ import {
 	Flex,
 	FlexItem,
 } from '@wordpress/components';
-import { check, plus, closeSmall } from '@wordpress/icons';
+import { check, plus, closeSmall, rotateRight } from '@wordpress/icons';
 import classnames from 'classnames';
 import { useTranslate } from 'i18n-calypso';
-import { useState, useCallback } from 'react';
+import { useState, useCallback, useEffect } from 'react';
+import { useScheduledUpdatesVerifyPathQuery } from 'calypso/data/plugins/use-scheduled-updates-verify-path-query';
+import { useSelector } from 'calypso/state';
+import getSiteId from '../../state/sites/selectors/get-site-id';
```

Comment:
Instead of using relative path here, maybe we can use `calypso` instead? `calypso/state/sites/selectors/get-site-id;`

Reply:
Agree! It was an auto-completed path, so I didn't see it. Replaced!

Thanks for the feedback.